modules/sdpolicy-am-compute.adoc

```diff
@@ -9,7 +9,10 @@ Single availability zone clusters require a minimum of 2 worker nodes for Custom
 
 Multiple availability zone clusters require a minimum of 3 worker nodes for Customer Cloud Subscription (CCS) clusters, 1 deployed to each of 3 availability zones. A minimum of 9 worker nodes are required for standard clusters. These 9 worker nodes are included in the base subscription, and additional nodes must be purchased in multiples of 3 to maintain proper node distribution.
 
-Worker nodes must all be the same type and size within a single {product-title} cluster.
+[NOTE]
+====
+Worker nodes must all be the same type and size within a single  {product-title} machine pool, but can be different types and sizes across multiple machine pools within an {product-title} cluster.
```

Comment:
I would suggest this reads a bit awkwardly. Possible suggestions:

All worker nodes within a single {product-title} machine pool must be of the same type and size. However, worker nodes across multiple machine pools within an {product-title} cluster can be of different types and sizes.

If the reordering does not work I would at least add the "of" in two places.

Reply:
Thanks @mramendi, I was not really happy with what I had but was drawing a blank, so I appreciate it!